src/main/kotlin/com/craftschemers/hub/commands/PrintPlayersInGameCommand.kt

```diff
@@ -0,0 +1,38 @@
+package com.craftschemers.hub.commands
+
+import com.craftschemers.hub.Hub
+import com.craftschemers.hub.minigame.Minigame
+import org.bukkit.ChatColor
+import org.bukkit.command.CommandSender
+
+object PrintPlayersInGameCommand : ICommand {
+
+    override val name = "print"
+    override val aliases = null
+    override val canBeConsole = true
+    override val description = "Use this command to print players in minigames (for debug purposes)"
+    override val parameters = null
+    override val usage = arrayOf("/hub print")
+    override val permissionMessage = "Permission message"
+    override val permission = "permission"
+
+    override fun onCommand(sender: CommandSender, minigame: Minigame?, label: String, args: Array<String>): Boolean {
+        val gameManager = Hub.plugin.gameManager
+        for (value in gameManager.minigames.values) {
```

Comment:
```suggestion
        gameManager.minigames.values.forEach { value ->
```
Maybe change `value` to `minigame` or something as well.